client/src/components/ViewModeToggle/ViewModeToggle.scss

```diff
@@ -0,0 +1,67 @@
+.viewModeToggle {
+  margin-left: auto;
+
+  .viewMode-selected.btn {
+    &,
+    &:hover,
+    &:before,
+    &.highlighted,
+    &:focus {
+      background: $dropdown-link-active-bg;
+      color: $component-active-color;
+      border: 0;
+    }
+  }
+
+  #splitModeButton {
+    &:disabled {
+      background: red;
+      pointer-events: all;
+    }
+  }
+}
+
+.popover-container:before {
+  content: none;
+}
+
+.disabled-tooltip {
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  height: 37px;
+  background: rgba(255, 255, 255, 0);
+
+  &::before {
+    content: "Screen size too small";
+    text-align: center;
+    background: #555;
+    color: white;
+    padding: 4px 0;
+    position: absolute;
+    top: -24px;
+    left: -1px;
+    right: -1px;
+    border-radius: 3px 3px 0 0;
+    display: none;
+    z-index: 2;
+  }
+
+  &::after {
+    content: "";
+    position: absolute;
+    width: 10px;
+    height: 10px;
+    transform: rotate(45deg);
+    display: none;
+    background: #555;
+    left: 47%;
+    top: -6px;
```

Comment:
if you implement the `span` suggestion above, you can then use borders to implement the "speech bubble" effect which will reduce a lot of this juggling around with css3 rotate and other specific pixel placements.